server/services/v1/database/query_runner.go

```diff
@@ -550,7 +558,7 @@ func (runner *StreamingQueryRunner) iterateOnKvStore(ctx context.Context, tx tra
 			// pass it to filterable
 			iter, err = reader.FilteredRead(iter, options.filter)
 		}
-	} else if iter, err = reader.ScanTable(options.table); err == nil {
+	} else if iter, err = reader.ScanPrimaryKeyTable(options.primaryKeyFrom); err == nil {
```

Comment:
I think calling it ScanTable is still fine as the scan table in this context is still valid.